soc/riscv/riscv-privilege/telink_b91/CMakeLists.txt

```diff
@@ -1,6 +1,10 @@
 # Copyright (c) 2021 Telink Semiconductor
 # SPDX-License-Identifier: Apache-2.0
 
+if (CONFIG_TELINK_B91_USB_SWI_ENABLE)
+	message(WARNING "USB SWI interface is enabled")
```

Comment:
May be INFO is better otherwise it seems like something we should fix

Reply:
Done

Reply:
STATUS         = Incidental information

Reply:
I can do so, BUT it is then not so visible in builds logs:

-- Found assembler: /home/jankokry/workspace/Telink/zephyr_sdk/zephyr-sdk-0.15.2/riscv64-zephyr-elf/bin/riscv64-zephyr-elf-gcc
-- USB SWI interface is enabled
-- Configuring done
-- Generating done

It is treated as a simple text. Should I proceed with this ?

Reply:
Changed to STATUS.